### PR TITLE
added a book for graphics programming

### PR DESCRIPTION
**Pull request by ltwmori**
## Hacktoberfest notes:

- due to volume of submissions, we may not be able to review PRs that do not pass tests and do not have informative titles.
- please read our [contributing guidelines](/CONTRIBUTING.md)
- be sure to check the output of Travis-CI for linter errors
- if this is your first open source contribution, make sure it's not your last!

## What does this PR do?
Add Resource(s) | Remove Resource(s) | Add info | Improve Repo

### Why is this valuable (or not)?
It's valuable because teaches the basics of 3d modeling in blender (very popular free and open source 3D creation suite)
### How do we know it's really free?
Blender is free, so its official manual is free too. 
### For book lists, is it a book? For course lists, is it a course? etc.
It is a book 
### Checklist:
- [x] Not a duplicate
- [x] Included author(s) if appropriate
- [x] Lists are in alphabetical order
- [x] Needed indications added (PDF, access notes, under construction)

### CODE REVIEW

File: free-programming-books.md [hunk 1]
```diff
@@ -274,7 +274,8 @@
 ### Graphics Programming
 
 * [3D Game Shaders For Beginners](https://github.com/lettier/3d-game-shaders-for-beginners) - David Lettier (Git) [(HTML)](https://lettier.github.io/3d-game-shaders-for-beginners)
-* [Blender 3D: Noob to Pro](https://en.wikibooks.org/wiki/Blender_3D%3A_Noob_to_Pro) - Wikibooks
+* [Blender 3D: Noob to Pro](https://en.wikibooks.org/wiki/Blender_3D%3A_Noob_to_Pro) (Wikibooks)
```
Comment (SethiPandi) [inline free-programming-books.md:277]: Could you revert this, I believe Wikibooks was meant to be separated with a hyphen.
Please see other resources from Wikibooks on this page for examples.

```suggestion
* [Blender 3D: Noob to Pro](https://en.wikibooks.org/wiki/Blender_3D%3A_Noob_to_Pro) - Wikibooks
```

File: free-programming-books.md [hunk 2]
```diff
@@ -274,7 +274,8 @@
...
+* [Blender Manual](https://docs.blender.org/manual/en/latest) (Wikibooks)
```
Comment (SethiPandi) [inline free-programming-books.md:278]: This isn't on Wikibooks.

```suggestion
* [Blender Manual](https://docs.blender.org/manual/en/latest)
```

**Review by SethiPandi — CHANGES_REQUESTED** [review]
Just left a few comments for you!

You can get more guidance on the formatting and conventions in this repository by checking out the [contribution guide](https://www.github.com/EbookFoundation/free-programming-books/tree/master/CONTRIBUTING.md).

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**